File: java/src/main/java/model/Process.java
package model;

public class Process {
    private String name;
    private int arrivalTime;
    private int duration;
    private int priority;
    private int completionTime;
    private int turnAroundTime;
    private int waitingTime;

    public Process() {
    }

    public int getArrivalTime() {
        return arrivalTime;
    }

    public void setArrivalTime(int arrivalTime) {
        this.arrivalTime = arrivalTime;
    }

    public int getDuration() {
        return duration;
    }

    public void setDuration(int duration) {
        this.duration = duration;
    }

    public int getCompletionTime() {
        return completionTime;
    }

    public void setCompletionTime(int completionTime) {
        this.completionTime = completionTime;
    }

    public int getTurnAroundTime() {
        return turnAroundTime;
    }

    public void setTurnAroundTime(int turnAroundTime) {
        this.turnAroundTime = turnAroundTime;
    }

    public int getWaitingTime() {
        return waitingTime;
    }

    public void setWaitingTime(int waitingTime) {
        this.waitingTime = waitingTime;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getPriority() {
        return priority;
    }

    public void setPriority(int priority) {
        this.priority = priority;
    }
}

File: python/model/Process.py
class Process:

  def __init__(self, name, duration, priority, arrival_time):
    self.name = name
    self.duration = duration
    self.priority = priority
    self.arrival_time = arrival_time
    self.completion_time = 0
    self.turn_around_time = 0
    self.waiting_time = 0

  def __str__(self):
    return f'{self.name} - {self.duration} - {self.priority} - {self.completion_time} - {self.arrival_time}'


  def set_waiting_time(self, waiting_time):
    self.waiting_time = waiting_time

  def get_waiting_time(self):
    return self.waiting_time